### ISSUE TITLE
Experiment requests should be ordered

### ISSUE THREAD

**Issue by guseggert** [ipfs-shipyard/thunderdome#99]
IIUC, each req gets its own SQS queue. Standard queues are [sometimes ordered, but not always](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/standard-queues.html#standard-queues-message-order). To preserve order when replaying traffic, we should use [FIFO queues](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html) (or Kinesis streams?). This would also prevent duplicate messages.

**Comment by iand** [comment]
All targets in an experiment get the requests in the same order since `dealgood` reads from the queue sequentially and sends that request to each target (or drops it if the target is overloaded). So in a single experiment all nodes see the same sequence of requests. However another independent experiment may see a different ordering.

**Comment by guseggert** [comment]
Yes but SQS queues themselves do not guarantee order unless they are FIFO queues.
